plugins/org.eclipse.n4js.ide/src/org/eclipse/n4js/ide/xtext/server/XBuildManager.java

```diff
@@ -141,10 +141,12 @@ public String toString() {
 	public List<IResourceDescription.Delta> doInitialBuild(List<ProjectDescription> projects,
 			CancelIndicator indicator) {
 
-		List<ProjectDescription> sortedDescriptions = sortByDependencies(projects);
+		IOrderInfo<ProjectDescription> orderInfo = projectBuildOrderProvider.get(projects);
```

Comment:
Renaming the orderInfo to orderedProjects would slightly improve the readability below